packages/zengine-ui/package.json

```diff
@@ -7,16 +7,18 @@
   "scripts": {
     "start": "npm run sass -- --watch",
     "watch": "npm run sass -- --watch",
-    "sass": "sass scss/style.scss lib/style.css",
-    "sass:min": "sass scss/style.scss lib/style.min.css --style compressed",
-    "build": "rimraf lib; npm run sass; npm run sass:min; cpy scss lib --parents; cpy package.json README.md CHANGELOG.md lib",
```

Comment:
@tehpsalmist we can remve rimraf and cpy from dev deps

Reply:
I thought I did

Reply:
maybe i missed the diff no worries thx